autodrive_devkit.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e
+
+# Setup development environment
+source /opt/ros/foxy/setup.bash
+source /home/projects/f1tenth_ws/autodrive_devkit/install/setup.bash
```

Comment:
Please put `autodrive_devkit/` under ` src/external/f1tenth_sim/autodrive_devkit/`
It should not be tracked by git